tests/system/model_monitoring/test_model_monitoring.py

```diff
@@ -330,9 +333,10 @@ def test_basic_model_monitoring(self, with_sql_target: bool) -> None:
             )
             sleep(choice([0.01, 0.04]))
 
-        sleep(5)
+        sleep(15)
         endpoints_list = mlrun.get_run_db().list_model_endpoints(
-            self.project_name, metrics=["predictions_per_second"]
+            self.project_name,
+            metrics=[mm_constants.EventLiveStats.PREDICTIONS_PER_SECOND],
```

Comment:
delete this as well

Reply:
why?

Reply:
there is no `metrics ` in the new version of `list_model_endpoints()` method